tardis/io/util.py

```diff
@@ -12,6 +13,34 @@
 import logging
 logger = logging.getLogger(__name__)
 
+float_pattern = r"""    # A numeric string consists of:
+    (?P<sign>[-+])?              # an optional sign, followed by either...
+
+    (
+
+        (?=\d|\.\d)              # ...a number (with at least one digit)
+
+        (?P<int>\d*)             # having a (possibly empty) integer part
+
+        (\.(?P<frac>\d*))?       # followed by an optional fractional part
+
+        (E(?P<exp>[-+]?\d+))?    # followed by an optional exponent, or...
+
+    |
+
+        Inf(inity)?              # ...an infinity, or...
```

Comment:
It is, the regex translates to "inf" optionally followed by "inity". If you are talking about the case, we use this pattern with re.IGNORECASE.